README.md

```diff
@@ -71,13 +71,42 @@ c.close()
 from confluent_kafka import avro
 from confluent_kafka.avro import AvroProducer
 
-value_schema = avro.load('ValueSchema.avsc')
-key_schema = avro.load('KeySchema.avsc')
+
+value_schema_str = """
```

Comment:
While this is somewhat verbose it is indeed nice with a self-contained examply :+1: